package.json

```diff
@@ -10,9 +10,9 @@
   "scripts": {
     "clean": "rm -rf allure-results && rm -rf allure-report",
     "test": "npm run clean && wdio run wdio.conf.js",
-    "test:local": "npm run clean && wdio run wdio.local.conf.js",
-    "test:local:debug": "DEBUG=true npm run test:local",
-    "test:docker": "npm run clean && wdio run wdio.docker.conf.js",
+    "test:local": "npm run clean && BASE_URL=\"http://localhost:3000\" wdio run wdio.local.conf.js",
+    "test:local:debug": "npm run clean && DEBUG=true BASE_URL=\"http://localhost:3000\" wdio run wdio.local.conf.js",
+    "test:docker": "npm run clean && BASE_URL=\"http://cdp.127.0.0.1.sslip.io:3333\" wdio run wdio.local.conf.js",
```

Comment:
I was hoping we could remove the `:docker` one and just use the `:local` one listening to an env var if set otherwise default to localhost. That way we could also push this to the template and let tenants override it in an env var if they want to